jsapp/js/account/usage/usageProjectBreakdown.tsx

```diff
@@ -112,33 +79,91 @@ const ProjectBreakdown = () => {
     setOrder(newOrder)
   }
```

Comment:
why bother an identity function, just use `setOrder` directly instead of `updateOrder`?

Reply:
I don't know, I didn't write this code. I don't want to change old code if I'm editing an existing file so I usually just leave the logic as is. Perhaps it was originally supposed to hold more logic here, but Ruth never got to it. Do you think we should simplify this?